.gitignore

```diff
@@ -106,4 +106,7 @@ dist
 
 jest_html_reporters.html
 
-reports
\ No newline at end of file
+reports
+
+helm/values.yaml
+helm/values.yml
```

Comment:
why did we switch from having values yaml, to not having one?
a chart cant work without one, so it adds another step.

Reply:
fixed